packages/framework-types/src/config.ts

```diff
@@ -105,6 +105,16 @@ export class BoosterConfig {
   }
 
   public get provider(): ProviderLibrary {
+    if (this._provider) {
+      console.warn(`
+      The usage of the 'config.provider' field is deprecated,
+      please use 'config.providerPackage' instead.
+
+      For more information, check out the docs:
+
+      https://docs.booster.cloud/chapters/05_going-deeper?id=configuration-and-environments
+      `)
+    }
```

Comment:
Wouldn't this error appear every time we want to get the provider in Booster core?

If I understood this properly, this message should go (without the if) in the setter. Correct?

Reply:
Correct! Nice catch, thanks!!